starter-code/css/main.css

```diff
@@ -0,0 +1,114 @@
+/**********************
+ * Author: Matt Ray
+ * Created: 9/13/17
+ * Links to: 'starter-code/index.html'
+ ***********************/
+
+ body {
+ 	font-family: 'Lato', sans-serif;
+ 	width: 960px;
```

Comment:
Rather than using a px value here, try using a % so you can keep your site responsive.